src/internal/m365/graph/errors.go

```diff
@@ -70,6 +70,7 @@ const (
 	NoSPLicense                     errorMessage = "Tenant does not have a SPO license"
 	parameterDeltaTokenNotSupported errorMessage = "Parameter 'DeltaToken' not supported for this request"
 	usersCannotBeResolved           errorMessage = "One or more users could not be resolved"
+	siteCouldNotBeFound             errorMessage = "Requested site could not be found"
```

Comment:
nit: trying to keep the names and errors as similar as possible, just in case we need to differentiate between closely matched variations in the future.
```suggestion
	requestedSiteCouldNotBeFound             errorMessage = "Requested site could not be found"
```